packages/bundle-source/src/tool.js

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env node
+import '@endo/init';
+import { ZipReader } from '@endo/zip';
+import { decodeBase64 } from '@endo/base64';
+import { makeReadPowers } from '@endo/compartment-mapper/node-powers.js';
+
+import { isEntrypoint } from './is-entrypoint.js';
+import bundleSource from './index.js';
+
+const { details: X, quote: q } = assert;
+
+const USAGE =
+  'bundle-source dest/ module1.js bundleName1 module2.js bundleName2 ...';
```

Comment:
consider space-delimiting the pairs and `:` delimiting its elements,
```suggestion
  'bundle-source dest/ bundleName1:module1.js bundleName2:module2.js  ...';
```

Reply:
Pro: delimited terms make it easier to add a third optional term for the cache location.
Con: can’t use `xargs -n 2 bundle-source`.

Reply:
hm. colons don't feel right... I think they would get in the way of tab-complete on filenames.

I narrowed the amount of CLI space that I'm claiming by requiring a leading `--to` arg. I'll let others who feel more creative do their thing.